.github/workflows/_test_te.yaml

```diff
@@ -51,20 +69,61 @@ jobs:
           pip install pytest-reportlog
           pytest --report-log=/log/report.jsonl ${SRC_PATH_TE}/tests/jax
 
+      - name: Generate sitrep
+        if: success() || failure()
+        shell: bash -x -e {0}
+        run: |
+          # bring in utility functions
+          source .github/workflows/scripts/to_json.sh
+
+          badge_label='TE Unit test'
+          failed_tests=$(cat /log/report.jsonl | grep -c 'failed' || true)
+          passed_tests=$(cat /log/report.jsonl | grep -c 'passed' || true)
+          total_tests=$((failed_tests + passed_tests))
+          
+          if [[ ${errors} > 0 ]] || [[ ${total_tests} == 0 ]]; then
+            badge_message='error'
+            badge_color=red
+            summary='TE Unit test did not complete due to errors.'
+          else
+            badge_message="${passed_tests}/${total_tests} passed"
+            if [[ ${failed_tests} == 0 ]]; then
+              badge_color=brightgreen
+            else
+              badge_color=yellow
+            fi
+            summary="TE Unit test: $badge_message"
+          fi
+          echo "failed tests: " $failed_tests " passed tests: " $passed_tests " total_tests: " $total_tests " summary: " $summary
+          to_json \
+            summary \
+            errors total_tests passed_tests failed_tests \
+            badge_label badge_color badge_message \
+          > /log/sitrep.json
+
+          schemaVersion=1 \
+          label="${badge_label}" \
+          message="${badge_message}" \
+          color="${badge_color}" \
+          to_json schemaVersion label message color \
+          > /log/$te-unit-test.json
```

Comment:
Is that dollar sign reference intentional?

Reply:
fixed the typo.